tests/test_create_company.py

```diff
@@ -0,0 +1,26 @@
+''''''
```

Comment:
what is this?

Reply:
It was supposed to be a docstring but I did not finish it and I don't know why. I will fix it